lib/rspec/request_describer.rb

```diff
@@ -70,5 +70,11 @@ def self.included(base)
         end
       end
     end
+
+    # @private
+    def self.process_with_kwargs?
+      ActionDispatch::Integration::Session.instance_method(:process) ||
+      ActionDispatch::Integration::Session.instance_method(:process_with_kwargs) rescue false
```

Comment:
Because actionpack 4.x also has `#process` method, I think it always returns truthy value, even if it does not support keyword arguments in `#process` method.